ui/webui/src/components/review/ReviewConfiguration.jsx

```diff
@@ -82,22 +71,20 @@ const checkDeviceInSubTree = (device, rootDevice, deviceData) => {
 };
 
 const DeviceRow = ({ deviceData, disk, requests }) => {
-    const [isExpanded, setIsExpanded] = useState(false);
-
     const data = deviceData[disk];
     const name = data.name.v;
 
     const renderRow = row => {
-        const iconColumn = row.reformat.v ? <CheckCircleIcon /> : null;
-        return {
-            props: { key: row["device-spec"] },
-            columns: [
-                { title: row["device-spec"] },
-                { title: row["format-type"] },
-                { title: row["mount-point"] },
-                { title: iconColumn },
-            ]
-        };
+        const format = row["format-type"] ? cockpit.format(_("format as $0"), row["format-type"]) : null;
+        const mount = row["mount-point"] || null;
+        const reformat = row.reformat ? _("reformat") : null;
+        const actions = [format, mount, reformat].filter(Boolean).join(", ");
+
+        return (
```

Comment:
I noticed that the mockup includes the size of the partitions. This is a not a regression from the previous state, so it can be followup.